### PR TITLE
Handle html responses

### PR DESCRIPTION
**Pull request by matthewrsj**
#70

Handle HTTP responses that do not have JSON response bodies.
* Put ``json.loads`` in a try - except block, format an error message in the except
* Rename ``_json_to_python()`` to ``_response_to_python()`` and pass it the entire response so it can handle all types

### CODE REVIEW

**Comment by MaraJade** [comment]
+1

File: pymesync.py [hunk 1]
```diff
@@ -357,9 +357,16 @@ def _local_auth_error(self):
...
-        """Convert json object to native python list of objects"""
-        python_object = json.loads(str(json_object))
+    def _response_to_python(self, response):
+        """Convert response to native python list of objects"""
+        try:
+            python_object = json.loads(str(response.text))
+        except ValueError:
+            # If we get a ValueError, response.text isn't a JSON object, and
+            # therefore didn't come from a TimeSync connection.
+            error_message = "your baseurl {0} returned status {1}".format(
```
Comment (Kennric) [inline pymesync.py]: I think I would re-word this. If we get a 200 (a normal page, just not timesync, or someone sends a 404 page with 200 status), it's going to be confusing to see 'error, your baseurl returned 200.' Maybe 'connection to timesync failed at baseurl {0}. Response status was: {1}'

**Comment by Kennric** [comment]
+1

**Comment by MaraJade** [comment]
+1

**Pull request merged**